api/nodes/views.py

```diff
@@ -2382,6 +2382,8 @@ class NodeProvidersList(JSONAPIBaseView, generics.ListAPIView, NodeMixin):
     view_category = 'nodes'
     view_name = 'node-providers'
 
+    ordering = ('-pk',)
```

Comment:
Just curious: any reason this is different than the other views that do `('-id',)`?